app/config/config_test.yml

```diff
@@ -7,6 +7,8 @@ parameters:
     api_url_public: http://api.elifesciences.org
     api_mock: '%kernel.cache_dir%/guzzle/fixtures'
     crm_url: http://crm.elifesciences.org/crm/civicrm/
+    feature_can_always_use_hypothesis: '%env(FEATURE_CAN_ALWAYS_USE_HYPOTHESIS)%'
+    env(FEATURE_CAN_ALWAYS_USE_HYPOTHESIS): false
```

Comment:
I guess this is the default for the environment variable (from https://symfony.com/doc/current/configuration/external_parameters.html)

Reply:
Yep.